pkg/token/athenz.go

```diff
@@ -19,11 +19,14 @@ import (
 	"crypto/x509"
 	"fmt"
 	"net/http"
+	"net/url"
 	"os"
+	"strconv"
+	"strings"
 
 	"github.com/AthenZ/athenz/clients/go/zts"
-	"github.com/AthenZ/athenz/libs/go/athenzutils"
 	"github.com/AthenZ/k8s-athenz-sia/v3/third_party/log"
+	"gopkg.in/square/go-jose.v2/jwt"
```

Comment:
```suggestion
	"gopkg.in/square/go-jose.v2/jwt"
	jwt "github.com/golang-jwt/jwt/v5"
```
can we use the existing library?